tests/acceptance/10_files/12_acl/acl_fail.cf

```diff
@@ -0,0 +1,67 @@
+# These tests don't work
```

Comment:
This file `acl_fail.cf` do you want to add it to this commit set? Then you might want to give it a different name, since it doesn't fail any more. And I you probably forgot to delete the commented policy at the top.